docs/source/man.rst

```diff
@@ -137,14 +137,21 @@ To create a credential, supply SSH credentials as either a username-password pai
 
   Prompts for the passphrase to be used when connecting using an ssh keyfile that requires a passphrase. Can only be used with the ``--sshkeyfile`` option.
 
-``--sudo-password``
+``--become_password``
 
-  Prompts for the password to be used when running a command that uses sudo on the systems to be scanned.
+  Prompts for the privilege escalation password to be used when running an ansible playbook.
 
+``--become_method=become_method``
 
-The information in a credential, such as a password, sudo password, SSH keys, or even the username, might change. For example, network security might require passwords to be updated every few months. Use the ``qpc cred edit`` command to change the SSH credential information in a credential. The parameters for ``qpc cred edit`` are the same as those for ``qpc cred add``.
+  Sets the method to become for privilege escalation when running an ansible playbook. Must be ``sudo``, ``su``, ``pbrun``, ``pfexec``, ``doas``, ``dzdo``, ``ksu``, ``runas``. The default is set to 'sudo' when the credential type is network.
 
-**qpc cred edit --name=** *name* **--username=** *username* **(--password | --sshkeyfile=** *key_file* **)** **[--sshpassphrase]** **[--sudo-password]**
+``--become_user=user``
+
+  Sets the user to become when running an ansible playbook. The default is set to 'root' when the credential type is network.
```

Comment:
`Sets the user to become when running an ansible playbook.`  --> `Sets the user to become when running a privileged command during network scan.`